src/python/pants/backend/python/goals/pytest_runner_integration_test.py

```diff
@@ -270,6 +270,27 @@ def test_ignore_me():
     assert "collected 2 items / 1 deselected / 1 selected" in result.stdout
 
 
+def test_xdist_enabled(rule_runner: RuleRunner) -> None:
+    rule_runner.write_files(
+        {
+            f"{PACKAGE}/tests.py": dedent(
+                """\
+                def test_run_me():
+                  pass
+
+                def test_run_me_2():
+                  pass
+                """
+            ),
+            f"{PACKAGE}/BUILD": "python_tests()",
+        }
+    )
+    tgt = rule_runner.get_target(Address(PACKAGE, relative_file_path="tests.py"))
+    result = run_pytest(rule_runner, tgt, extra_args=["--pytest-xdist-enabled=True"])
+    assert result.exit_code == 0
+    assert "2 passed" in result.stdout
```

Comment:
How do we know that xdist was used? Would this test pass if you changed the above line to False?

Reply:
I think to really test this I'll need to split out separate tests for interactive and non-interactive cases - that way I can assert on the presence of the env vars xdist injects in the non-interactive case, and assert on their absence in the interactive case.

Reply:
At a minimum, it would be great to have a test for the non-interactive case to check for the env vars. Added bonus if you're able to test interactive case :)

Reply:
As mentioned on the ticket, I think that this is fine as a coverage test. But if you see a reasonable way to do it, then by all means.